tests/end-to-end/e2e/fixtures_queries.go

```diff
@@ -11,13 +11,18 @@ import (
 )
 
 //Application
-func createApplicationFromInputWithinTenant(t *testing.T, ctx context.Context, gqlClient *gcli.Client, tenant string, in graphql.ApplicationCreateInput) graphql.ApplicationExt {
+func createApplicationWithinTenant(t *testing.T, ctx context.Context, gqlClient *gcli.Client, tenant string, in graphql.ApplicationCreateInput) (graphql.ApplicationExt, error) {
```

Comment:
I would move this to your test case to private method.
Because here, we only create resources successfully.
And I think that error is a part of your logic and it should be there, but maybe I am wrong.

Reply:
Hmm, but it would mean that I need to duplicate code - as this is not `_test` file. I will keep it as it is. I will move it to the bottom of this file.